common/src/main/java/com/wynntils/services/lootrunpaths/LootrunPathsService.java

```diff
@@ -186,10 +186,10 @@ public void tryLoadLootrun(String fileName) {
 
             BlockPos start = PosUtils.newBlockPos(startingPoint);
             McUtils.sendMessageToClient(Component.translatable(
-                            "feature.wynntils.lootrunUtils.lootrunStart", start.getX(), start.getY(), start.getZ())
+                            "service.wynntils.lootrunpaths.lootrunStart", start.getX(), start.getY(), start.getZ())
                     .withStyle(ChatFormatting.GREEN));
         } else {
-            McUtils.sendErrorToClient(I18n.get("feature.wynntils.lootrunUtils.lootrunCouldNotBeLoaded", fileName));
+            McUtils.sendErrorToClient(I18n.get("service.wynntils.lootrunpaths.lootrunCouldNotBeLoaded", fileName));
```

Comment:
Should this be `lootrunPaths` for consistency? I think everything else is camelCase

Reply:
You're absolutely correct. I'll fix.